examples/examples_switch_cases.cpp

```diff
@@ -40,7 +40,7 @@ int main() {
 		std::size_t input = 100;
 		auto res = switch_cases<0, 20>(
 				input,
-				[](auto i) { return fib_v<i>; },
+				[]<auto i>() { return fib_v<i>; },
```

Comment:
Note the interface change here, I think this makes it more obvious that the param is a const expression